data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -219,6 +219,29 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# Maryland EJSCREEN Data.
+MARYLAND_EJSCREEN_TRACT_25_PERCENTILE_FIELD: str = (
+    "Tract is >=25% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_50_PERCENTILE_FIELD: str = (
+    "Tract is >=50% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_75_PERCENTILE_FIELD: str = (
+    "Tract is >=75% all other Maryland Tracts"
+)
+MARYLAND_EJSCREEN_TRACT_90_PERCENTILE_FIELD: str = (
+    "Tract is >=90% all other Maryland Tracts"
+)
+
+MARYLAND_EJSCREEN_SCORE_FIELD: str = "Maryland Environmental Justice Score"
+
+# this references percentile ranks
+# please see here for interpretation: https://github.com/usds/justice40-tool/issues/239#issuecomment-1003567593
+# and here: https://github.com/usds/justice40-tool/issues/239#issuecomment-1003448074
+MARYLAND_EJSCREEN_BURDENED_THRESHOLD_FIELD: str = (
+    "Tract has a percentile rank above 75 for the EJ Score"
```

Comment:
As previously discussed, please call these percentiles and do not call them percentile rank. Also try to match the format you've used for similar field names throughout `field_name`.

Reply:
I will remove this.